bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/chart/ChartProvider.java

```diff
@@ -54,6 +54,8 @@ public interface ChartProvider {
      * @param items The items to display on the chart
      * @param groups The groups to display on the chart
      * @param dpi The DPI (dots per inch) value, can be <code>null</code>
+     * @param interpolation The interpolation between two datapoint (<code>linear</code> or <code>step</code>),
+     *            <code>null</code> defaults to <code>linear</code>)
```

Comment:
Does this contradict what you wrote in the issue description?

>  If not set, the current behaviour will be used (step for OnOff or OpenClosed and linear for all other values).

Reply:
Yes, indeed. I clarified it.